tests/cpu_bugs/xen_domu_mitigation_test.pm

```diff
@@ -701,22 +701,48 @@ sub install_domu {
               . " --vnc"
               . " --events on_reboot=destroy"
               . " --serial pty", timeout => 5400);
+        if ($status =~ /Domain is still running. Installation may be in progress/) {
+            diag "Guest installation successful.\n";
+
+        } else {
+            diag "Error:guest installation,Please check the status of guest.\n";
+            return 0;
+        }
     }
+
     # Check if the DomU is up
     script_run("virsh start \"${domu_name}\"");
+    return 1;
 }
 
+sub verifyIp {
```

Comment:
As we talk, this function should be added to get_guest_ip.sh, right?

Reply:
I have deleted this function and add some codes in get_guest_ip.sh, Please help me review this logic.